app/main.py

```diff
@@ -2,7 +2,7 @@ def copy_file(command: str) -> None:
     command_parts = command.split(" ")
     file_name = command_parts[1]
     copy_file_name = command_parts[2]
-    if command_parts[0] != "cp" or len(command) != 3:
+    if command_parts[0] != "cp" or len(command_parts) != 3:
```

Comment:
you must do it before` file_name = command_parts[1]`, because if command doesn't have 3 arguments code will be broken